site/src/App.tsx

```diff
@@ -64,17 +71,19 @@ export const AppProviders: FC<AppProvidersProps> = ({
 	}, []);
 
 	return (
-		<HelmetProvider>
-			<QueryClientProvider client={queryClient}>
-				<AuthProvider>
-					<ThemeProvider>
-						{children}
-						<GlobalSnackbar />
-					</ThemeProvider>
-				</AuthProvider>
-				{showDevtools && <ReactQueryDevtools initialIsOpen={showDevtools} />}
-			</QueryClientProvider>
-		</HelmetProvider>
+		<CacheProvider value={cache}>
```

Comment:
could provider actually go inside `ThemeProvider` instead, since it's already doing emotion related stuff?

Reply:
yes, seems like that is fine.